### DISCUSSION TITLE
v7.13.16 wasn't retrievable as 'latest' version for as long as it was the latest (24h).

### DISCUSSION THREAD

**Discussion by dbeezt** [babel/babel#13189] [General] [answered]
I noticed that for the duration of time where v7.13.16 was the latest-available package version, running 'pnpm install' failed to retrieve v7.13.16. Instead, it could retrieve the previous version - v7.13.14. With the update to v7.13.17 at approx. midnight last night, the latest package retrievable was obtained as v7.13.17. I want to know whether v7.13.16 was missing some metadata information, or for some other reason wasn't obtainable as the latest version? 

I've had a look at the release notes of v7.13.16 and v7.13.17 but haven't found anything to indicate that: https://github.com/babel/babel/releases.

**Comment by nicolo-ribaudo** [comment, marked as answer]
Please report this bug to npm :pray: 

It's really annoying, this happens quite often, but once the package is on the registry there is nothing that we can do since it's an npm or proxy bug.

**Reply by dbeezt** [reply]
Will do, and I'm glad to hear it's not package-specific. Thaks for the speedy reply!
